packages/uicore/src/components/MultiStepProgressIndicator/MultiStepProgressIndicator.css

```diff
@@ -58,3 +58,17 @@
 .spacing {
   padding: 2px !important;
 }
+.dotNameSuccess {
+  margin-top: -26px;
```

Comment:
Can we not use negative values here? Negative value for `margin-top` could be replaced with a `margin-bottom` instead.